source/Octopus.Tentacle.Tests.Integration/Support/TentacleConfigurationsAttribute.cs

```diff
@@ -53,9 +63,22 @@ static class TentacleConfigurationTestCases
             bool testScriptIsolationLevel,
             bool testDefaultTentacleRuntimeOnly,
             ScriptServiceVersionToTest scriptServiceToTest,
+            bool testPolling,
+            bool testListening,
             object[] additionalParameterTypes)
         {
-            var tentacleTypes = new[] { TentacleType.Listening, TentacleType.Polling };
+            var tentacleTypes = new List<TentacleType>();
+
+            if (testPolling)
+            {
+                tentacleTypes.Add(TentacleType.Polling);
+            }
+
+            if (testListening)
```

Comment:
Nit: Do you think it's worth ensuring we've added at least one of `testPolling` or `testListening`?

Reply:
100% - Update pushed